src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
```

Comment:
Prefer using 'const' over 'let' for 'result' since it's not being reassigned, but rather its value is being updated in a new scope each time.